### DISCUSSION TITLE
openwrt: unable to use untagged vlan on LAN during setup

### DISCUSSION THREAD

**Discussion by cbrunnkvist** [tteck/Proxmox#3109] [Help Needed]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

When you try set up a VM using the `openwrt` script in "Advanced" mode, there is no clear way to make LAN interface untagged and it ends up defaulting to id 999.

VM creation fails in my case, as I intend to attach LAN to a Proxmox Simple VNet where vlan:s are prohibited/unavailable as a feature.

### What settings are you currently utilizing?

Advanced Settings

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

```
...
Using LAN Vlan: 999
...
```
_a few lines later..._
```
 - Starting OpenWrt VM...vm vlans are not allowed on vnet leetnet0 at /usr/share/perl5/PVE/Network/SDN/Zones/Plugin.pm line 235.
kvm: -netdev type=tap,id=net0,ifname=tap140i0,script=/var/lib/qemu-server/pve-bridge,downscript=/var/lib/qemu-server/pve-bridgedown,vhost=on: network script /var/lib/qemu-server/pve-bridge failed with status 6400
start failed: QEMU exited with code 1

[ERROR] in line 516: exit code 255: while executing command qm start $VMID
```

### Please provide detailed steps to reproduce the issue.

Run `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/vm/openwrt.sh)"`, select `Advanced`,
then there are two prompts related to vlan:s:

1) vlan for WAN, prompt says "leave empty for untagged" or something like that
2) vlan for LAN, the prompt default to 999 but if you delete that, assuming it would work like the previous prompt,

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/1961
